### ISSUE TITLE
White page loading a dashboard: Can not find path "models.propertyDefinitionsModel" in the store

### ISSUE THREAD

**Issue by macobo** [PostHog/posthog#7119]
## Bug description

Try to open https://app.posthog.com/dashboard/20201?highlightInsightId=507967 

The following error pops up:

```
instrument.ts:129 Error: [KEA] Can not find path "models.propertyDefinitionsModel" in the store.
    at /static/chunk-54TPR5DD.js:6
    at Array.reduce (<anonymous>)
    at OGe (/static/chunk-54TPR5DD.js:6)
    at e.reducer.e.selector (/static/chunk-54TPR5DD.js:6)
    at /static/chunk-54TPR5DD.js:5
    at Object.propertyStorage (/static/chunk-54TPR5DD.js:5)
    at Object.get [as propertyStorage] (/static/chunk-54TPR5DD.js:6)
```

## Expected behavior

Page loads

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

## Additional context

cc @mariusandra

**Comment by macobo** [comment]
Weird. Now it loads after failing to for a while. Didn't see any chunk errors in the browser console either.

Note this error is also not showing up in sentry - wonder what that's about?

**Comment by mariusandra** [comment]
This is fixed in #7124, but is not the real error. Something's probably causing React to crash earlier, which unmounts the logics. I had a case with an out-of-order dayjs import. Moving the call to dayjs.utc inside a function to be lazily evaluated helped.

Dayjs has been pretty annoying TBH with esbuild. Could be do something? We still have `moment` in the codebase, so I'd just use that... or whatever :D.
